src/main/java/org/wso2/carbon/connector/pcml/AS400Constants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.connector.pcml;
+
+import org.wso2.carbon.utils.CarbonUtils;
+
+import java.io.File;
+
+/**
+ * Constants for the connector.
+ */
+public class AS400Constants {
+    // Properties stored in the message context.
+    public static final String AS400_INSTANCE = "as400Instance";
+    public static final String AS400_SOCKET_PROPERTIES = "as400SocketProperties";
+    public static final String AS400_CONNECTION_POOL_MAP = "as400ConnectionPoolMap";
+    // Parameters of the initializing mediator.
+    public static final String AS400_INIT_SYSTEM_NAME = "systemName";
```

Comment:
Fixed in https://github.com/wso2-extensions/esb-connector-pcml/pull/3/files#diff-aa3a15281a8a00341109fa1ef26ab435R30